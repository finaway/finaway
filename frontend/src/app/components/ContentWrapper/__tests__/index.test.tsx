import * as React from 'react';
import { render } from '@testing-library/react';

import { ContentWrapper } from '..';

describe('<ContentWrapper  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <ContentWrapper>Hello World</ContentWrapper>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });

  it('should render the children', () => {
    const { getByText } = render(<ContentWrapper>Hello World</ContentWrapper>);
    expect(getByText('Hello World')).toBeInTheDocument();
  });
});
