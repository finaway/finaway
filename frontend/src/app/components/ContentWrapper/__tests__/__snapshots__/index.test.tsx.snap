// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentWrapper  /> should match snapshot 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthSm css-cuefkz-MuiContainer-root"
>
  Hello World
</div>
`;
